docs/acme-protocol-updates.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: ACME Protocol Updates
+permalink: /docs/acme-protocol-updates/
+top_graphic: 1
+---
+
+[<- Back to Documentation List](/docs/)
+
+The ACME protocol is the cornerstone of how Let's Encrypt works. As the protocol specification evolves over time Let's Encrypt will implement updated versions of ACME. When doing so, security will be our primary concern, followed closely by backwards compatibility.
+
+# New Backwards-Compatible ACME Features
+
+From time to time Let's Encrypt may implement new backwards-compatible features for existing API endpoints. Typically new backwards-compatible features are introduced because we've decided to implement a part of the ACME spec that we hadn't implemented before.
+
+When new features are introduced to existing API endpoints, the features will always be clearly specified in a public ACME specification and will not break properly implemented clients.
+
+# ACME Security Fixes
+
+If we become aware of a serious security issue with the ACME protocol (rather than simply our implementation of it) we may be forced to make compatibility-breaking changes to our API endpoints, or to cease operation of existing endpoints and introduce new ones. Hopefully this never happens, but this possibility highlights why it's important that ACME clients be able to receive timely updates.
```

Comment:
Rather than "Hopefully," I would say: "ACME has been reviewed by many parties and used successfully in production, but there is always the possibility of undiscovered vulnerabilities. Systems administrators should maintain the ability to deploy timely updates to their ACME clients in response to such vulnerabilities."